Firstbranch

randallwvr90 opened the pull request:
Hey!

This is a pull request which means I made a branch and now I think we should merge that branch with the main branch. The main branch is where the "real" code lives and new branches are an easy way to try new code. Then once you are happy with the new code, you merge your branch with the main branch. I added you as a reviewer so you can look and we can discuss the change before merging with the main branch. 

Anyway, I just made two files: musicplayer and drumplayer. They only have comments so far with a high-level description of what they would do. Do you think this is a good way to do things?

Commit:
initial commit for musicplayer.java

Commit:
Initial commit for drumplayer and updated musicplayer

I called the thing that would play the drums musicplayer.java but realized that musicplayer should play music files. So I created drumplayer and changed musicplayer accordingly. This is also a chance to try a multi-line commit message from the shell.

JimWeaverOK commented on ux/musicplayer.java:
```diff
@@ -0,0 +1,13 @@
+	- play_drums
+		- it could have some kind of global variable that will increase or decrease the volume and/or speed. 
+		  Maybe one for each? volumen and speed?
+		- Will it play drums for a certain amount of time? Will it play drums until it gets some other input?
```
I image some patterns or cadences, too

JimWeaverOK commented on ux/musicplayer.java:
```diff
@@ -0,0 +1,13 @@
+	What kind of methods does it need?
+	- play_drums
+		- it could have some kind of global variable that will increase or decrease the volume and/or speed. 
+		  Maybe one for each? volumen and speed?
```
volume and speed could be separate for max flexibility

JimWeaverOK commented on ux/musicplayer.java:
```diff
@@ -0,0 +1,13 @@
+	- change_volume
+	- change_speed
+*************************************************************************************************************************
+*/
```
I'm thinking of having the drums either respond to a green house gas calculation and also play at random intervals.  Intensity is determined by the greeenhouse calculator, or is determined randomly.

randallwvr90 replied:
I was thinking there will be some other object that "decides" what it should tell the output objects to do based on the greenhouse calculator. Or I guess the output objects could keep checking to see when the calculation changes and then change their behavior accordingly.

Commit:
Made some changes based on Jim's comments.

This is still comments only, with no functionality. It's just placeholder stuff. Added information about playing specific drum cadences and changing music in musicplayer.
